### PR TITLE
RFC 3: Book participation

### LINKED ISSUE #4: RFC Request: How Much Should Others Be Able to Contribute to the Book
@steveklabnik Thought I would formally document this here after our short conversation on IRC about how much others would be allowed to contribute new content to the book. You admitted (quite understandably) that you had not given it much thought as you weren't sure anyone would want to participate in helping you write the book. 

I noted that the RFC repo would be the perfect place for you to share your thoughts on these questions. Questions such as: Should others only be allowed to offer small corrections? Should others be allowed to flesh out already existing sections of the book? How about entirely new, previously unwritten sections? Should others have a say in what direction the kernel and therefore the book take?

Since you are the creator of this entire project, naturally it makes the most sense for you to make whatever decision you feel is most fair to you, but in the spirit of open development, it would be interesting to see your thoughts written out.

### PR DESCRIPTION
**Pull request by steveklabnik**
Fixes #4 

@rylev what do you think? Also, I added you as part of the initial editorial team, is that okay with you?

### CODE REVIEW

**Comment by dariusk** [comment]
It might be worth specifically mentioning the licensing/copyright of the contributed content. "The work contributed by authors is [ownership policy goes here]." Just pointing to a LICENSE.md or something would probably suffice.

**Comment by rylev** [comment]
@steveklabnik This looks really great. I would be very happy to be a part of the editorial team. :smile: 

I'm hesitant though to have a role that is officially equal to yours. You having final say on "big decisions" (of course with feedback from other editors and the community at large) will go a long way in maintaining consistency of voice. However, I'm not exactly sure how we should define what constitutes a "big decision".

Is there any way we could define an "editor in chief" role which you would occupy?

**Comment by steveklabnik** [comment]
@dariusk ah, i'm normally so good with license stuff :( Yes, we should deal with this soon.

> Is there any way we could define an "editor in chief" role which you would occupy? 

Well, I'm always trying to balance how much hierarchy is actually necessary. But if you think that it's worthwhile, we could add it.

> However, I'm not exactly sure how we should define what constitutes a "big decision".

This is true. We could just take the easy way out and not define it, and rely on our judgement.

**Comment by rylev** [comment]
Ok, let's keep things simple and revisit this idea if we feel the need in the future. With such a small editorial team an informal understanding is more than enough. I just wanted to state for the record that I still recognize intermezzOS as _your_ project, and I trust your judgement when it comes to making the "big decisions" (despite the fact that, as I said, I don't even know what I mean with that term). I'm looking forward to helping out as much as I can!

File: text/0003-book-participation.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+I would like for the book to not be solely written by myself. That said, when
+I started this project, I wasn’t sure if anyone would even want to help out,
+and I had a plan. Including others means that plan may change.
+
+Specifically, here is my proposal:
+
+* There are two groups of contributors to the book: editors and authors.
+* All editors are authors, but not all authors are editors.
+* Editors have a commit bit, authors must send in PRs.
+* Initially, Steve will be the sole editor, but if others are interested, we’ll
```
Comment (rylev) [inline text/0003-book-participation.md]: We should probably change this to reflect the two person editorial team mentioned below.

Comment (steveklabnik) [inline text/0003-book-participation.md]: oh yeah hahah

nice job catching consistency there :+1: :smile:

**Comment by steveklabnik** [comment]
I'm going to merge this RFC, and create a new RFC for the licensing issue, as we should discuss what that license shoudl be.

**Pull request merged**